src/zui/ZUIPersonGridEditCell.tsx

```diff
@@ -0,0 +1,288 @@
+import { Close } from '@mui/icons-material';
+import { makeStyles } from '@mui/styles';
+import { useRouter } from 'next/router';
+import { Msg, PlainMessage, useMessages } from 'core/i18n';
+
+import messageIds from './l10n/messageIds';
+import { usePersonSelect } from './ZUIPersonSelect';
+import { ZetkinPerson } from 'utils/types/zetkin';
+import ZUIAvatar from 'zui/ZUIAvatar';
+
+import {
+  Box,
+  Button,
+  InputBase,
+  List,
+  ListItem,
+  ListSubheader,
+  Paper,
+  Popper,
+  Theme,
+  Typography,
+  useAutocomplete,
+} from '@mui/material';
+import { FC, HTMLAttributes, useState } from 'react';
+
+const ZUIPersonGridEditCell: FC<{
+  cell?: (Partial<Omit<ZetkinPerson, 'id'>> & { id: number | null }) | null;
+  onUpdate: (person: ZetkinPerson | null) => void;
+  removePersonLabel: PlainMessage;
```

Comment:
I don't think there is any other case in the code where a message is being passed into a component like this. Pass in the string instead?

I.e. when using the component instead of doing this:

```tsx
<ZUIPersonGridEditCell removePersonLabel={messageIds.some.message}/>
```

Do this:

```tsx
const messages = useMessages(messageIds);
<ZUIPersonGridEditCell removePersonLabel={messages.some.message()}/>
```

This is better for many reasons, one of which is that it's the responsibility of the user of the component to deal with things like interpolation etc. The way it's been hard-coded now, only plain messages will work.

Reply:
I was wondering about best practices here. Good.